osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/entities/DlmsDevice.java

```diff
@@ -56,9 +50,6 @@
     @Column
     private boolean hls5Active;
 
-    @OneToMany(mappedBy = "dlmsDevice", fetch = FetchType.EAGER, cascade = { CascadeType.ALL }, orphanRemoval = true)
-    private final List<SecurityKey> securityKeys = new ArrayList<>();
```

Comment:
Dropping the security keys from DlmsDevice is in line with the intentions behind using the secret management service.

As there no longer is a use for `SecurityKey` in the platform It looks like the `SecurityKey` entity and the `security_key` table should be removed, and not left around without a purpose.

Reply:
Agreed.